### PR TITLE
Update Page for possible #JSCC17

### PR DESCRIPTION
**Pull request by davelosert**
Hey Guys,

I went ahead and updated the page to be less "2016" specific.
@marcoemrich @wolframkriesing @lowsky : What do you think?

### CODE REVIEW

**Comment by wolframkriesing** [comment]
I am getting this here, when trying to `yarn build` or `yarn start`

```
> yarn run build
yarn run v0.15.1
$ mkdir -p ./dist 
$ harp compile ./src ./dist; cp CNAME dist/; npm run 'build:participants.json' 
module.js:457
    throw err;
    ^

Error: Cannot find module 'promise'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/xxx/jsCraftCamp/website/node_modules/jstransformer/index.js:5:15)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

you guys too?

**Comment by davelosert** [comment]
@wolframkriesing : you `yarn` Version is really old - maybe try to update it to the current (_v.0.19.x_)

File: package.json [hunk 1]
```diff
@@ -15,9 +17,10 @@
     "prebuild": "mkdir -p ./dist",
     "build": "harp compile ./src ./dist; cp CNAME dist/; npm run 'build:participants.json'",
     "build:participants.json": "node scripts/concat-json.js ./participants dist/participants.json; cp ./dist/participants.json ./src/",
-    "start": "harp server ./src",
+    "start": "harp server ./src & browser-sync start --proxy 'localhost:9000' --files 'src/**/*.ejs, src/**/*.css', src/**/*.js",
```
Comment (lowsky) [inline package.json]: I would split this into 2 scripts:

* start : harp server...
* dev : yarn run start & browser-sync ...

File: README.md [hunk 2]
```diff
@@ -8,8 +8,8 @@ This project contains the http://jscraftcamp.org site.
 node.js 5.x
```
Comment (lowsky) [inline README.md]: minor: Could already be >6 or even >7 shouldn't it, right?

Comment (lowsky) [inline README.md]: We could mention yarn, and how to get / install it...

File: README.md [hunk 3]
```diff
@@ -8,8 +8,8 @@ This project contains the http://jscraftcamp.org site.
...
 
 ## Build
-- run `npm install` to install all dependencies
-- run `npm start` which will run the local webserver, for developing
-- run `npm test` to run the tests
-- run `npm run build` to generate all static files and to concatenate all participants files into the `dist` directory
+- run `yarn` to install all dependencies
```
Comment (lowsky) [inline README.md]: Nice.

**Pull request merged**